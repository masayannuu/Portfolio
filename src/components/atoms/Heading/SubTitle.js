import React from 'react'
import styled from 'styled-components'
import { Subtitle } from 'bloomer'

const StyledSubTitle = styled(Subtitle)`
  color: #a2c4c7;
`
const SubTitle = props => <StyledSubTitle isSize={2}>{props.children}</StyledSubTitle>

export default SubTitle
