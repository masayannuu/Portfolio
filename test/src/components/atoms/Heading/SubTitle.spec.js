import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { shallowToJson } from 'enzyme-to-json'

import SubTitle from '../../../../../src/components/atoms/Heading/SubTitle'

Enzyme.configure({ adapter: new Adapter() })

describe('Heading SubTitle Atoms', () => {
  it('should render SubTitle', () => {
    const wrapper = Enzyme.shallow(<SubTitle>subtitle</SubTitle>)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
