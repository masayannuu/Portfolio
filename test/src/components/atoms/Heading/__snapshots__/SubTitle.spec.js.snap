// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading SubTitle Atoms should render SubTitle 1`] = `
<SubTitle__StyledSubTitle
  isSize={2}
>
  subtitle
</SubTitle__StyledSubTitle>
`;
